src/coreclr/src/inc/corinfo.h

```diff
@@ -750,7 +751,8 @@ enum CorInfoUnmanagedCallConv
     CORINFO_UNMANAGED_CALLCONV_C,
     CORINFO_UNMANAGED_CALLCONV_STDCALL,
     CORINFO_UNMANAGED_CALLCONV_THISCALL,
-    CORINFO_UNMANAGED_CALLCONV_FASTCALL
+    CORINFO_UNMANAGED_CALLCONV_FASTCALL,
```

Comment:
Revert this change? The comment says "correspond to CorUnmanagedCallingConvention", but CorUnmanagedCallingConvention is not getting this added.